Dockerfile

```diff
@@ -1,3 +1,18 @@
-FROM starefossen/github-pages:onbuild
+# Get Jekyll build env
+FROM docs/docker.github.io:docs-builder AS builder
 
-CMD bundle update && bundle exec jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
+# Build the docs from this branch
+COPY . /source
+RUN jekyll build --source /source --destination /site
+
+# Reset to bare Nginx image
+FROM nginx:alpine
+
+# Copy the Nginx config from the docs-config image
+COPY --from=docs/docker.github.io:docs-config /conf/nginx-overrides.conf /etc/nginx/conf.d/default.conf
```

Comment:
@thaJeztah Is this the right way not to bust the cache due to a config change?

Reply:
Yes, because the nginx-configuration should not have an effect on the content of the HTML